botorch/models/multitask.py

```diff
@@ -417,6 +418,8 @@ def __init__(
             likelihood: A `MultitaskGaussianLikelihood`. If omitted, uses a
                 `MultitaskGaussianLikelihood` with a `GammaPrior(1.1, 0.05)`
                 noise prior.
+            data_covar_module: The module computing the covariance (Kernel) matrix
+            in data space. If omitted, use a `MaternKernel`.
```

Comment:
One. last. docstring. issue. Otherwise this is great, thanks for fixing this up!

```suggestion
                in data space. If omitted, use a `MaternKernel`.
```